test/test_simulation_results.jl

```diff
@@ -182,11 +193,11 @@ function test_simulation_results(file_path::String, export_path)
             @test size(param)[1] == 48
         end
 
-        realized_duals_ed = read_realized_duals(results_ed)
-        @test length(keys(realized_duals_ed)) == 1
-        for param in values(realized_duals_ed)
-            @test size(param)[1] == 576
-        end
+        #realized_duals_ed = read_realized_duals(results_ed)
```

Comment:
All these tests if they don't apply we can remove

Reply:
I think they do apply. It's just that duals are not stored. Is there a bug or test misconfiguration?